cmd/scollector/collectors/database.go

```diff
@@ -0,0 +1,345 @@
+package collectors
+
+import (
+	"database/sql"
+	"fmt"
+	"strconv"
+	"strings"
+	"sync"
+	"time"
+
+	_ "bosun.org/_third_party/github.com/go-sql-driver/mysql"
+	"bosun.org/cmd/scollector/conf"
+	"bosun.org/collect"
+	"bosun.org/metadata"
+	"bosun.org/opentsdb"
+	"bosun.org/slog"
+)
+
+type DatabaseQuery struct {
```

Comment:
They are not the same. How would you like to handle the time in the collectors.DatabaseQuery? Could convert the interval on each run if you want but the next seems more tricky.
	interval    time.Duration
	next        time.Time